docroot/sites/default/settings.php

```diff
@@ -208,3 +208,8 @@
   ini_set('memory_limit', '4096M');
 }
 
+// https://github.com/drush-ops/drush/issues/676#issuecomment-1136068584
+if (PHP_SAPI === 'cli') {
+  $databases['default']['default']['init_commands']['wait_timeout'] = 'SET SESSION wait_timeout = 3600';
```

Comment:
Especially after comparing the wait_timeout on Acquia vs our ddev environments, this seems promising